react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -4,55 +4,53 @@ import styles from '../../styles.module.scss';
 import Navbar from "../Navbar"
 import { connect } from "react-redux";
 import actionCreators from "../../../../../redux/actions";
+import Spinner from 'react-spinkit';
 
 class Matches extends Component {
   
   async componentDidMount(){
-    const response = this.props.dispatch(actionCreators.getMatches());
-    //this.props.isLoading = true;
-    //const response = await TicTacToeService.getMatches();
-    if(response.ok){
-      this.props.matches = response.data;
-      this.props.isLoading = false;
-    }
+    this.props.dispatch(actionCreators.getMatches());
   }
 
   render() {
     if(!window.localStorage.getItem("token")){
       return <Redirect to="/"/>;
     }
 
-    const matchesHistory = this.props.matches;
-    const matches = matchesHistory.map(game => {
-      const match = "Player 1: " + game.player_one + " || Player 2: " +  game.player_two + " || Winner: " + game.winner;
-      return (
-        <li key={game.id} >
-          {match}
-        </li>
-      );
-    });
-    var Spinner = require('react-spinkit');
+    const {matches} = this.props;
 
     if(this.props.isLoading){
-      return (<Spinner name='folding-cube' />);
-    }else{
-      return (
-        <div className={styles.game}> 
-          <Navbar secondScreen="game"/>
-          <div className={styles.gameInfo}>
-            <ol>{matches}</ol>
-          </div>
-        </div>
-      );
+      return <Spinner name='folding-cube' />;
+    }
+
+    if(this.props.error){
+      return <span>ERROR:  {this.props.error}</span>;
     }
+
+    return (
+      <div className={styles.game}> 
+        <Navbar secondScreen="game"/>
+        <div className={styles.gameInfo}>
+          <ol>{
+            matches.map(game => {
+              const match = "Player 1: " + game.player_one + " || Player 2: " +  game.player_two + " || Winner: " + game.winner;
```

Comment:
Again, I think removing this const will make this easier to read.
It just a style preference